public/translations/en-US.json

```diff
@@ -323,6 +324,7 @@
         "validation_on_detail": "the action will be available until a date, quantity or both",
         "verification_label": "Does it need verification?",
         "verifiers_label": "Choose the verifiers",
+        "verifiers_label_count": "Choose at least {{count}} verifiers",
```

Comment:
We are still missing translations for other languages. Usually we just use google translate, we had little errors with it. Currently we support `en-US`, `pt-BR`, `es-ES` and `cat-CAT`

here is the Portuguese one: `Escolha ao menos {{count}} verificadores`